### ISSUE TITLE
 Huawei AR651-X8 and Huawei AC6605[Missing SNMP Polling Profile]

### ISSUE THREAD

**Issue by Aldenxu** [kentik/snmp-profiles#346]
[NOTE]: # ( ^^ Please add the Device Vendor and Model in the title above. ^^ )

Device Vendor: Huawei

Device Model: Huawei AR651-X8

[NOTE]: # (If you do not know your SysOID, it can be gathered from the SNMP walk you provide at the end of this issue. It is the value of this OID in the SYSTEM-MIB: `1.3.6.1.2.1.1.2.0`)

System Object Identifier (SysOID): 1.3.6.1.4.1.2011.2.224.279

List of critical metrics that you expect in the profile (does not need to be all-inclusive, just a general idea of your expectations):

 - 
 -
 -

**Please provide a [public gist](https://gist.github.com/) with a sanitized** *(sensitive information removed)* **SNMP walk of the device. Providing MIB files is not a substitute for an SNMP walk and failure to provide one here will result in delays for closing this issue. The output of these commands shows the "true story" of what a device will respond to in SNMP and is used to ensure that our profiles are efficient and accurate for your devices.**

Gist Link: 

---
Example SNMP walk commands:
```
# v2c example
snmpwalk -v 2c -t 10 -On -c $COMMUNITY_STRING $IP_ADDRESS . >> snmp_walk.out

# v3 example
snmpwalk -v3 -t 10 -l authPriv -u $USERNAME -a MD5|SHA -A $AUTH_PASS -x DES|AES -X $PRIV_PASS -ObentU -Cc $IP_ADDRESS . >> snmp_walk.out
```

[Huawei Router snmpwalk-X.X.X.102.txt](https://github.com/kentik/snmp-profiles/files/8998723/Huawei.Router.snmpwalk-X.X.X.102.txt)
[Huawei Router snmpwalk-X.X.X.101.txt](https://github.com/kentik/snmp-profiles/files/8998724/Huawei.Router.snmpwalk-X.X.X.101.txt)

**Comment by Aldenxu** [comment]
You have done this before for us but we can only get 5% of the data and rest of them has nothing even missing match profile error disappeared. please reload this to see.

**Comment by Mesverrum** [comment]
@Aldenxu those walks you have included there are probably a bit overly sanitized for what we need.  Of particular interest to us is the results of 1.3.6.1.2.1.1.2, we use this value to map a specific device to the profile so we can't make any progress without it.  

It's possible that we may just need to add a collection of these sysObjectId's from all your similar models of Huawei routers to the profile @thezackm made last week.

**Comment by Aldenxu** [comment]
.1.0.8802.1.1.2.1.3.4.0 = STRING: "Huawei AR651-X8 Huawei Versatile Routing Platform Software  VRP (R) software,Version 5.170 (AR651 V300R003C00SPC500) Copyright (C) 2011-2018 Huawei Technologies Co., Ltd". please let me know this is what you need.

**Comment by Mesverrum** [comment]
Device included in profile #341
